### PR TITLE
chore(deps): upgrade node minor version, some packages

### PR DESCRIPTION
**Pull request by miketheman**
With the current packages, we can see some issues with the npm modules:

```console
$ docker compose run --rm static npm audit
...
found 314 vulnerabilities (97 moderate, 109 high, 108 critical) in 1812 scanned packages
  run `npm audit fix` to fix 239 of them.
  58 vulnerabilities require semver-major dependency updates.
  17 vulnerabilities require manual review. See the full report for details.
```

I tried to tackle a few here, see each commit for details descriptions.

As the end of this, we remain with:

```console
found 135 vulnerabilities (42 moderate, 53 high, 40 critical) in 2635 scanned packages
```

I'm game to continue and update/upgrade others, but wanted to ensure that this was the right direction before going further.

Signed-off-by: Mike Fiedler <miketheman@gmail.com>

### CODE REVIEW

File: tests/frontend/password_breach_controller_test.js [hunk 1]
```diff
@@ -43,6 +43,39 @@ describe("Password breach controller", () => {
   });
 
   describe("functionality", () => {
+  /*
+    This does not feel good right now, but will allow progress.
```
Comment (miketheman) [inline tests/frontend/password_breach_controller_test.js:47]: Reviewers - this is likely the most contentious change. Please read the notes - if there's another/better way to solve, this, I'd be super existed to learn about it!

**Comment by miketheman** [comment]
@di any thoughts here?
Also, is there anyone else who'd be a good javascript reviewer?

**Comment by di** [comment]
Much appreciated @miketheman!

**Comment by miketheman** [comment]
Thanks @di ! I'll follow this up with some more updates, and 🤞 we can safely move to stimulus 3.

**Pull request merged**